aws/provider.go

```diff
@@ -995,6 +995,7 @@ func Provider() *schema.Provider {
 			"aws_route53_resolver_query_log_config_association":       resourceAwsRoute53ResolverQueryLogConfigAssociation(),
 			"aws_route53_resolver_rule_association":                   resourceAwsRoute53ResolverRuleAssociation(),
 			"aws_route53_resolver_rule":                               resourceAwsRoute53ResolverRule(),
+			"aws_route53_recovery_readiness_cell":                     resourceAwsRoute53RecoveryReadinessCell(),
```

Comment:
See comments in #20367.

```suggestion
			"aws_route53recoveryreadiness_cell":                     resourceAwsRoute53RecoveryReadinessCell(),
```

Reply:
Will go through and make these changes then rev.